book_source/03_intermediate_users_guide/01_pecan_xml.Rmd

```diff
@@ -438,22 +597,27 @@ The following tags can be used for state data assimilation. More detailed inform
 * **end.date** : [required?] end date of the state data assimilation (in YYYY/MM/DD format)
 * **_NOTE:_** start.date and end.date are distinct from values set in the run tag because this analysis can be done over a subset of the run.
```

Comment:
@para2x @araiho @istfer separate from this PR could you check that the list of tags here and their descriptions are up-to-date. Fine to keep all the gory details on a separate page, but don't want this page to be wrong

Reply:
I can take care of that in Multi-site PR.  I don't use `sample.parameters` anymore and instead I added `adjustment` in the re-factored version.